### PR TITLE
Image generator for Fedora dist-git repos

### PR DESCRIPTION
**Pull request by phracek**
This commit adds cwt generator used by https://github.com/sclorg/betka.

cwt-generator generates dist-git sources from upstream repositories.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: Dockerfile.generator [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM fedora:rawhide
+ENV CWT_DIR=/tmp/container-workflow-tool
+RUN dnf install -y go-md2man \
+    make git python3-PyYAML \
+    python3-GitPython \
+    python3-requests-kerberos \
+    python3-bugzilla \
```
Comment (pkubatrh) [inline Dockerfile.generator:7]: Do we need these in Fedora? no bugzilla module, and iirc you worked around not having to use kerberos to access dist-git right?

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+.PHONY: test build-generator push-generator
 test: $(TESTS)
 
 $(TESTS):
 	PYTHONPATH=.:$$PYTHONPATH python3 -W ignore::DeprecationWarning $(TEST_DIR)/$@.py -v
+
+build-generator:
+	docker build --tag ${GENERATOR_IMAGE} -f Dockerfile.generator .
+
+push-generator:
```
Comment (pkubatrh) [inline Makefile:13]: Add a dependency on build-generator

File: Dockerfile.generator (hunk 1 above)
Comment (pkubatrh) [reply Dockerfile.generator:7]: cwt runs with `--disable-klist` anyway

Comment (pkubatrh) [reply Dockerfile.generator:7]: Though I guess we would have to import it conditionally now I think about it. Lets just remove `python3-bugzilla` then

**Comment by pkubatrh** [comment]
Otherwise lgtm

**Comment by phracek** [comment]
PR is update. Sorry, I don't have permissions to merge the PR.

**Comment by pkubatrh** [comment]
Needs one more rebase because of the other PR I merged

**Comment by phracek** [comment]
Rebased.

**Comment by pkubatrh** [comment]
Thanks! Merging

**Pull request merged**